src/java/org/apache/cassandra/tools/nodetool/GetInterDCStreamThroughput.java

```diff
@@ -18,16 +18,23 @@
 package org.apache.cassandra.tools.nodetool;
 
 import io.airlift.airline.Command;
-
+import io.airlift.airline.Option;
 import org.apache.cassandra.tools.NodeProbe;
 import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
 
-@Command(name = "getinterdcstreamthroughput", description = "Print the Mb/s throughput cap for inter-datacenter streaming in the system")
+@Command(name = "getinterdcstreamthroughput", description = "Print the Mb/s throughput cap for inter-datacenter streaming and entire SSTable inter-datacenter streaming in the system")
 public class GetInterDCStreamThroughput extends NodeToolCmd
 {
+    @SuppressWarnings("UnusedDeclaration")
+    @Option(name = { "-esst", "--print-entire-sstable-throughput" }, description = "Print entire SSTable streaming throughput")
+    private boolean printEntireSSTableThroughput;
+
     @Override
     public void execute(NodeProbe probe)
     {
-        probe.output().out.println("Current inter-datacenter stream throughput: " + probe.getInterDCStreamThroughput() + " Mb/s");
+        if (printEntireSSTableThroughput)
+            probe.output().out.println("Current entire SSTable inter-datacenter stream throughput: " + probe.getEntireSSTableInterDCStreamThroughput() + " Mb/s");
```

Comment:
If throttling is disabled with negative values, `probe.getEntireSSTableInterDCStreamThroughput` returns the negative value too, which could be confusing. How about "Current entire SSTable inter-datacenter stream throughput: unlimited"?